generators/app/index.js

```diff
@@ -18,8 +21,7 @@ module.exports = Generator.extend({
         'Welcome to the remarkable ' + chalk.red('PatternLabStarter') + ' generator! ' + this.pkg.version + '\nPlease be in the folder you want files in now.'
```

Comment:
`ParticleStarter` instead?

Reply:
This is still going to be pulling down PLS v9 for awhile.